### ISSUE TITLE
uvbeam fails to read CST files with proper-case headers and does not appear to handle directivities/e-field amplitudes in dBi

### ISSUE THREAD

**Issue by anguta** [RadioAstronomySoftwareGroup/pyuvdata#437]

**Comment by plaplant** [comment]
@bhazelton is this solved by having the yaml files to save metadata?

**Comment by bhazelton** [comment]
No, I don't think so. @anguta has a PR on this that we've stalled on and I have had some separate conversations with Nicholas and Eloy that might help with this. I think it comes down to understanding the conversion between the various CST outputs -- it's not just a simple unit change.
